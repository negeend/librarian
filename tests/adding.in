ADD MEMBER James Bob
add member Jimmothy Smith Frank Johnathan
aDd mEmBeR Gary
MEMBER 100000
MEMBER 100001
MEMBER 100002
ADD BOOK nonexistent.csv 111111
add book books.csv 123456
add book empty.csv 111111
ADD BOOK books.csv 111111
ADD BOOK books.csv 111111
exit
