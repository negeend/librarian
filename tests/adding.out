user: Success.

user: Success.

user: Success.

user: 100000: James Bob

user: 100001: Jimmothy Smith Frank Johnathan

user: 100002: Gary

user: No such file.

user: No such book in file.

user: No such book in file.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Book already exists in system.

user: Ending Library process.
